<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OhCalendar - Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>360</width>
     <height>100</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>Basic</string>
    </attribute>
    <widget class="QLineEdit" name="input1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>340</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Try &quot;623+457&quot;</string>
     </property>
    </widget>
    <widget class="QLabel" name="result1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>340</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>=0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>Date</string>
    </attribute>
    <widget class="QDateEdit" name="input2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>170</width>
       <height>22</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>10</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2024</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="timeSpec">
      <enum>Qt::TimeSpec::LocalTime</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="input3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="input4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="suffix">
      <string> day(s)</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QLabel" name="result2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>340</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>= January 1, 2024</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
